resources/css/style.css

```diff
@@ -45,9 +45,22 @@
 .title-get-involved {
   font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
   font-size: 5em;
-  font-weight: 100;
+  font-weight: 300;
```

Comment:
This won't be required as it will bring inconsistency in font-weight
So lets keep it as 100,
see
 
![screen shot 2017-02-05 at 5 00 06 pm](https://cloud.githubusercontent.com/assets/13018570/22625805/97027620-ebc4-11e6-96be-a12b943347ad.png)


![screen shot 2017-02-05 at 4 59 34 pm](https://cloud.githubusercontent.com/assets/13018570/22625794/8143b4f2-ebc4-11e6-8ea7-bce209cecc47.png)